game-core/src/main/java/games/strategy/engine/data/GameParser.java

```diff
@@ -515,13 +513,10 @@ private void parseInfo(final Node info) {
     data.setGameVersion(new Version(version));
   }
 
-  private void parseGameLoader(final Node loader) throws GameParseException {
-    final String className = ((Element) loader).getAttribute("javaClass");
-    final Object instance = getInstance(className);
-    if (!(instance instanceof IGameLoader)) {
-      throw newGameParseException("Loader must implement IGameLoader.  Class Name:" + className);
+  private void parseGameLoader(final Node loader) {
+    if (loader != null) {
+      log.log(Level.INFO, "Loader tag is being ignored");
```

Comment:
:+1: Was going to suggest this as soon as I read the PR description.  Only recommendation is it might be simpler to call `log.info(...)`.